lib/ruby-push-notifications/mpns.rb

```diff
@@ -0,0 +1,3 @@
+require 'ruby-push-notifications/mpns/mpns_connection'
+require 'ruby-push-notifications/mpns/mpns_notification'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.